packages/components/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
 
 -   `Modal`: Fix loss of focus when clicking outside ([#52653](https://github.com/WordPress/gutenberg/pull/52653)).
 
+### Internal
+
+-   `TabPanel`: Introduce a new version of `TabPanel` with updated internals while maintaining the same functionality and API surface   ([#52133](https://github.com/WordPress/gutenberg/pull/52133)).
```

Comment:
Should we frame this as an enhancement, now that it adheres to the ARIA guidance better with the `tabpanel` focus behavior?